examples/workflow.yaml

```diff
@@ -14,7 +14,7 @@ imports:
   type: git
   parameters:
     repository: "git@github.com:zapatacomputing/z-quantum-qubo.git"
-    branch: "dev"
+    branch: "176770493-save-sampleset-to-json"
```

Comment:
don't forget to change this when merging! 
btw, I have a GitHub Actions bot to automatically change this when merging a PR. It's in the templates repo I think
 https://github.com/zapatacomputing/orquestra-resource-template

Reply:
Good point!

Reply:
Fixed here: https://github.com/zapatacomputing/z-quantum-qubo/pull/7/commits/aa026286ae8f709456dd36bee6e57e52fba3e286